src/components/ChallengeModal/ChallengeModal.test.tsx

```diff
@@ -145,7 +60,8 @@ jest.mock("@/lib/data", () => ({
         if (key === "preferred-game-settings") {
             return [];
         }
-        return null;
+
+        return default_value;
```

Comment:
`default_value` helps a lot, when most of the `get` calls specify it.  Adding this means we don't need to mock `TimeControlUpdates` or add an entry to the mock impl